### ISSUE TITLE
2.6 not support SqlCipher

### ISSUE THREAD

**Issue by xiongcaixing1986** [ccgus/fmdb#459]
#import <sqlite3.h> can not support SqlCipher

**Comment by ccgus** [comment]
We don't support SqlCipher, but we can try and get things working again.
Can you go into detail what's messed up?

**Comment by zhengfenwei** [comment]
old version has    #import "sqlite3.h"   in FMDatabase.h

**Comment by robertmryan** [comment]
Correct, so you now need to import this yourself. Unfortunately, to use FMDB in frameworks (which you need for Carthage and/or extensions), we can no long reference non-modular headers inside the public headers.

**Comment by xiongcaixing1986** [comment]
Thanks

**Comment by johnryan** [comment]
@robertmryan can you clarify the changes with frameworks? I just updated my project to use_frameworks and modules with cocoapods and now everything locks up on [db setKey:"key"]. Everything compiles fine so i'm a little confused as to what changed. Thanks.

**Comment by repl-ullas-ml** [comment]
There is a dead lock on [db setKey:"key"] if you choose use_frameworks. If you comment use_frameworks [db setKey:"key"] would not work
